### ISSUE TITLE
Add Sitemap

### ISSUE THREAD

**Issue by simonech** [Shazwazza/Articulate#193]
Add Google Sitemap for helping with indexing of the blog

**Comment by biapar** [comment]
Why this? I added the sitemap to website with Articulate blog. http://www.volaresereni.it/sitemap

**Comment by simonech** [comment]
Yeah, it can be easily added manually but it would be good to add it to the core.
It would be cool if you could send a PR :)

**Comment by biapar** [comment]
Yes. I used the good package CM Umbraco Tools where you can exclude some type of pages from sitemap https://our.umbraco.org/projects/developer-tools/cm-umbraco-tools/ . 
The only problem that I see with Articulate is that if you link to authors, I return the page not found. Others main issue is we don't have the tags and categories into the sitemap because they are not a really pages. So I think that we need to specialize the CM Umbraco Tools ( good think ) or build a package for Articulate.

**Comment by simonech** [comment]
Indeed, a custom site map would be needed inside articulate to take care of tags/categories

**Comment by Shazwazza** [comment]
Have marked as `help wanted`. Since sitemaps for sites can be generated in a ton of ways, perhaps this implementation can generate one for what Articulate knows about along with an API to do so so that if people are generating a sitemap for their entire site then can just use this API to get the Articulate part and merge it into their own. I'm open to any ideas but don't have time to get this done myself.

**Comment by biapar** [comment]
Specs?

**Comment by frederiktoft** [comment]
Just want to chime in and hope that a friendly soul will help out with this feature :)
